src/axom/mint/docs/sphinx/sections/getting_started.rst

```diff
@@ -8,29 +8,61 @@
 Getting Started with Mint
 ---------------------------
 
-This section provides a quick introduction to Mint by presenting a
-complete walk-through of an example Mint application, illustrating some of the
-key concepts and capabilities. The complete :ref:`MintApplicationCodeExample`,
-presented herein, is included in the :ref:`sections/mint/appendix` section and is
-also available in the Axom source code under
-``src/axom/mint/examples/mint_uniform_mesh.cpp``.
+This section presents a complete code walk-through of an example Mint
+application, designed to illustrate some of the key concepts and capabilities
+of Mint. The complete :ref:`MintApplicationCodeExample`, is included in the
+:ref:`sections/mint/appendix` section and is also available in the Axom source
+code under ``src/axom/mint/examples/user_guide/mint_getting_started.cpp``.
 
-The example Mint application illustrates:
-
-* Construction of a simple 2D :ref:`UniformMesh`
-* Creation of node-centered and cell-centered :ref:`FieldData` on the mesh
-* Using the :ref:`sections/execution_model` to implement *mesh-agnostic*
-  and *device-agnostic* kernels.
-* Output of the resulting mesh and data in VTK for visualization.
+The example Mint application demonstrates how Mint is used to develop
+kernels that are both *mesh-agnostic* and *device-agnostic*.
 
 .. tip::
+
     Mint's `RAJA`_-based :ref:`sections/execution_model` helps facilitate
     the implementation of various computational kernels that are both
     *mesh-agnostic* and *device-agnostic*. Both kernels discussed in
     this example do not make any assumptions about the underlying mesh
     type or the target execution device, e.g. CPU or GPU. Consequently, the
-    same implementation can operate on any of the supported :ref:`MeshTypes`
-    and run in parallel on all execution devices supported through `RAJA`_.
+    same implementation can operate on both :ref:`StructuredMesh` and
+    :ref:`UnstructuredMesh` types and run sequentially or in parallel on all
+    execution devices supported through `RAJA`_.
+
+Prerequisites
+^^^^^^^^^^^^^^
+
+Understanding the discussion and material presented in this section requires:
+
+* Working knowledge of C++ templates and `Lambda Expression`_ functions.
+* High-level understanding of nested memory hierarchies on heterogeneous
+  architectures and `CUDA Unified Memory`_.
+* Basics of the :ref:`sections/execution_model` (recommended)
+
+.. note::
+
+  Mint does not provide a memory manager. The application must explicitly
+  specify the desired allocation strategy and memory space to use. This is
+  facilitated by using `Umpire`_ for memory management. Specifically, this
+  example uses `CUDA Unified Memory`_ when compiled with `RAJA`_ and CUDA
+  enabled by setting the default `Umpire`_ allocator in the beginning of the
+  program as follows:
```

Comment:
It might be good to say something about what happens when umpire/raja/cuda are not available.

Reply:
Done.